### PR TITLE
Added Card for trello website

### PR DESCRIPTION
**Pull request by aishwarya-borke**
- Added cards for trello

### CODE REVIEW

File: src/components/cardListItems.tsx [hunk 1]
```diff
@@ -15,56 +22,85 @@ export default function WorkSpace() {
     setItems(null);
   };
 
-  const [text, setText] = useState('');
-  const handleChange = (event: { target: { value: React.SetStateAction<string>; }; }) => {
-    setText(event.target.value);
-  }
+  const [text, setText] = useState("");
+  const [card, setCard] = useState<any[]>([]);
```
Comment (anandhu-ks) [inline src/components/cardListItems.tsx]: why type any is used here ?

Comment (aishwarya-borke) [reply src/components/cardListItems.tsx]: Resolved

File: src/components/ListItem.tsx [hunk 2]
```diff
@@ -1,18 +1,98 @@
...
-    );
-}
\ No newline at end of file
+
+export default function ListItem(props: { title: string }) {
+  const [handlecard, setHandleCard] = useState(false);
+  const cardHandler = () => {
+    setHandleCard(true);
+  }
+  const [cardlist, setCardList] = useState("");
```
Comment (anandhu-ks) [inline src/components/ListItem.tsx]: this variable is used for card title right ? please give a proper name

File: src/components/ListItem.tsx [hunk 3]
```diff
@@ -1,18 +1,98 @@
...
+  const [listcd, setListCd] = useState<cardbox[]>([]);
+
+  const buttonClick = () => {
```
Comment (anandhu-ks) [inline src/components/ListItem.tsx:28]: rename the function

File: src/components/ListItem.tsx (hunk 2 above)
Comment (aishwarya-borke) [reply src/components/ListItem.tsx]: resolved

File: src/components/ListItem.tsx (hunk 3 above)
Comment (aishwarya-borke) [reply src/components/ListItem.tsx:28]: Resolved

File: src/components/cardListItems.tsx [hunk 4]
```diff
@@ -15,56 +22,87 @@ export default function WorkSpace() {
...
-  const [text, setText] = useState('');
-  const handleChange = (event: { target: { value: React.SetStateAction<string>; }; }) => {
-    setText(event.target.value);
-  }
+  const [text, setText] = useState("");
+  const [card, setCard] = useState<CardItems[]>([]);
 
-  const [addcard, setAddCard] = useState(false);
+  const handleClick = () => {
+    if (text.trim() !== '') {
```
Comment (anandhu-ks) [inline src/components/cardListItems.tsx]: you can use text.length !==0  here

File: src/components/ListItem.tsx [hunk 5]
```diff
@@ -1,18 +1,101 @@
...
+export default function ListItem(props: { title: string }) {
+  const [handlecard, setHandleCard] = useState(false);
+  const cardHandler = () => {
+    setHandleCard(true);
+  }
+  const [title, setTitle] = useState("");
+  const [titlelist, setTitleList] = useState<cardbox[]>([]);
+
+  const buttonClick = () => {
+    if(title.trim() !== '') {
```
Comment (anandhu-ks) [inline src/components/ListItem.tsx]: use title.trim.length

Comment (aishwarya-borke) [reply src/components/ListItem.tsx]: trim.length is no working... not getting output with using title.trim.length !== 0

File: src/components/cardListItems.tsx (hunk 4 above)
Comment (aishwarya-borke) [reply src/components/cardListItems.tsx]: resolved

**Review by anandhu-ks — APPROVED** [review]

**Pull request merged**